curriculum/challenges/english/25-front-end-development/quiz-javascript-problem-solving-and-algorithmic-thinking/66edd4bc0ce399f475ad6f9b.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The Fibonacci sequence is a series where each number is the sum of the two preceding ones, starting with `0` and `1`. The sequence goes: `0, 1, 1, 2, 3, 5, 8, 13, 21...` and so on. How can the nth number in this sequence be calculated?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use the formula `fib(n) = fib(n-1) * fib(n-2)`.
 
 ---
 
-Placeholder distractor 2
+Use recursion with memoization to avoid recalculating subproblems where `fib(n) = (n-1) + (n-2)`.
 
 ---
 
-Placeholder distractor 3
+Use a for loop to calculate iteratively where `fib(n) = fib(n-1) + (n-2)`.
 
 #### --answer--
 
-Placeholder answer
+Use recursion with memoization to avoid recalculating subproblems where `fib(n) = fib(n-1) + fib(n-2)`.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+A binary string, such as `"1011"`, represents a number in base-2, where each digit corresponds to a power of `2`. What process can be used to convert this binary string into its decimal equivalent?
 
 #### --distractors--
 
-Placeholder distractor 1
+Multiply each character by powers of `2` and sum them up.
```

Comment:
as the question is "what process can be used" this is also a correct answer

Reply:
Actually, answer should be "Multiply each character `(It's ASCII value - '0')` by powers of 2 and sum them up". I added this question to mislead whoever is taking the quiz.

Reply:
![image](https://github.com/user-attachments/assets/cac65320-eec7-4aae-b618-a40cbebe6ce0)
No, you really can just multiply the characters and let JS do the coercion.

Reply:
Thanks for pointing it out. Didn't know about coercion concept. Will replace the choice.